packages/algob/src/types.ts

```diff
@@ -652,3 +652,9 @@ export interface AnyMap {
 export type PromiseAny = Promise<any>; // eslint-disable-line @typescript-eslint/no-explicit-any
 
 //  LocalWords:  configFile
+
+export interface DebuggerContext {
```

Comment:
do we have this type in algosdk ? is it exported ?

Reply:
this isn't algosdk type. we're in algob/types. Custom params we can pass in a debugging session (`.run({ .. })`). Explained in docs.